src/main/scala/org/ua/ilm/Tube/YouTubeCrawlerChannels.scala

```diff
@@ -0,0 +1,18 @@
+package org.ua.ilm.Tube
+
+import java.io.File
+
+import com.google.api.services.youtube.model.ChannelListResponse
+import org.ua.ilm.FileChange.{ReadIdChannels, ReadPart}
+
+class YouTubeCrawlerChannels(partFile: File = new File("/Users/ekaterina/IdeaProjects/YTAPI/part.txt"),
+                     listFile: File = new File("/Users/ekaterina/IdeaProjects/YTAPI/channelsId.txt"))
```

Comment:
Not sure about the idea of writing such data in a file
Mb it would be better to use some collections